src/improvements/template/UnpauseSuperchainConfigV400.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {VmSafe} from "forge-std/Vm.sol";
+import {stdToml} from "forge-std/StdToml.sol";
+
+import {L2TaskBase} from "src/improvements/tasks/types/L2TaskBase.sol";
+
+import {Action} from "src/libraries/MultisigTypes.sol";
+
+/// @title UnpauseSuperchainConfigV400 template designed to unpause the SuperchainConfig contract after the U16 upgrade.
+contract UnpauseSuperchainConfigV400 is L2TaskBase {
+    using stdToml for string;
+
+    /// @notice Identifier of the eth_lockbox used in the SuperchainConfig
+    /// @dev Address (20 bytes) that has an associated lockbox contract, loaded from TOML
+
+    address public identifier;
+
+    /// @notice SuperchainConfig SC contract instance
+    ISuperchainConfig public sc;
+
+    /// @notice Returns the string identifier for the safe executing this transaction.
+    function safeAddressString() public pure override returns (string memory) {
+        return "Guardian";
+    }
+
+    /// @notice Returns string identifiers for addresses that are expected to have their storage written to.
+    function _taskStorageWrites() internal pure override returns (string[] memory) {
+        string[] memory storageWrites = new string[](1);
+        storageWrites[0] = "SuperchainConfig";
+        return storageWrites;
+    }
+
+    /// @notice Sets up the template with implementation configurations from a TOML file.
+    function _templateSetup(string memory _taskConfigFilePath, address _rootSafe) internal override {
+        super._templateSetup(_taskConfigFilePath, _rootSafe);
+        string memory file = vm.readFile(_taskConfigFilePath);
+        identifier = vm.parseTomlAddress(file, ".identifier"); // Get the identifier of the eth_lockbox from the TOML file
+        // 1. Load the SuperchainConfig contract.
+        sc = ISuperchainConfig(superchainAddrRegistry.get("SuperchainConfig"));
+    }
+
+    /// @notice Write the calls that you want to execute for the task.
+    function _build(address) internal override {
+        // 2. Make sure that the SuperchainConfig is paused before unpausing for the identifier.
+        assertEq(sc.paused(identifier), true, "ERR99: SuperchainConfig should be Paused for the identifier provided.");
+
+        // 3. UnPause the SuperchainConfig contract through the identifier.
+        sc.unpause(identifier);
+    }
+
+    /// @notice This method performs all validations and assertions that verify the calls executed as expected.
+    function _validate(VmSafe.AccountAccess[] memory, Action[] memory, address) internal view override {
+        // Validate that the SuperchainConfig contract is unpaused.
+        // 4. check that the Superchain Config is not paused anymore with the identifier provided.
+        assertEq(
+            sc.paused(identifier), false, "ERR100: SuperchainConfig should be unpaused for the identifier provided."
```

Comment:
Why does it start at ERR99 and increment from there? I think there are other patterns where we use codes throughout the code base but it's normally in test files iirc.